cli/release.ts

```diff
@@ -49,13 +52,24 @@ export async function handleRelease({
     try {
       const bundleFilename = await build(manifestFile);
       const manifest = await importManifest<PackVersionDefinition>(bundleFilename);
-      packVersion = manifest.version as string;
+      packVersion = manifest.version as string | undefined;
```

Comment:
Nit, this is my fault, but maybe these lines would be clearer as something like:

```
const manifest = await importManifest<BasicPackDefinition | PackVersionDefinition>(bundleFilename);
packVersion = 'version' in manifest && manifest.version;
```

Reply:
Ah, did you make `importManifest` templated in your last PR? Sorry, I must've missed that.